bin/git-mpush

```diff
@@ -165,11 +167,20 @@ sub run {
       error("Could not fast-forward branch '$slower_branch' to '$faster_branch'.");
     }
 
-    info("Local branches '$master_branch' and '$release_branch' are at the same point now.");
+    # checkout the slower branch
```

Comment:
comment should say checkout postreleasefix/branch and probably add a bit more to say we are fastforwarding the postreleasefix branch now